docker/graalvm/Dockerfile

```diff
@@ -0,0 +1,41 @@
+
+FROM oracle/graalvm-ce:20.1.0-java11
+
+RUN adduser --home /opt/besu besu && \
```

Comment:
Oracle linux is different, am I losing security by nor finding the replacement for `--disabled-password --gecos "" `?

Reply:
They are required on the other image to avoid having it prompt for the password and user information (like Full Name, Room Number etc).  Oracle Linux doesn't do those prompts so won't need the extra options.